### PR TITLE
fix fastapi instrumentation tests for version 0.91

### PR DESCRIPTION
**Pull request by Akochavi**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

tox -e test-instrumentation-fastapi

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py [hunk 1]
```diff
@@ -110,10 +112,7 @@ def test_uninstrument_app(self):
         self._client.get("/foobar")
         spans = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans), 3)
-        # pylint: disable=import-outside-toplevel
-        from fastapi.middleware.httpsredirect import HTTPSRedirectMiddleware
 
-        self._app.add_middleware(HTTPSRedirectMiddleware)
         self._instrumentor.uninstrument_app(self._app)
         print(self._app.user_middleware[0].cls)
```
Comment (srikanthccv) [inline instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py:117]:
```suggestion
```

**Review by srikanthccv — COMMENTED** [review]
Was there a reason behind adding `HTTPSRedirectMiddleware` in `test_uninstrument_app`?

File: instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py (hunk 1 above)
Comment (Akochavi) [reply instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py:117]: Removed

**Comment by Akochavi** [comment]
@srikanthccv According to what I checked, there was no specific reason to add it there.

**Review by srikanthccv — APPROVED** [review]

**Review by shalevr — APPROVED** [review]

**Comment by phillipuniverse** [comment]
Fixes #1665 as a follow-up to #1664

**Pull request merged**